requirements.txt

```diff
@@ -15,8 +15,8 @@ mirakuru==0.9.0
 netifaces==0.10.7
 networkx==2.1.0
 psutil==5.4.5
-raiden-libs==0.1.4
-raiden-contracts==0.2.0
+git+https://github.com/andrevmatos/raiden-libs@8232994067d5472faa4502047292ce488569a596#egg=raiden-libs
```

Comment:
This you will change to normal raiden libs once https://github.com/raiden-network/raiden-libs/pull/125/f is merged I guess?

Reply:
Yes, exactly